bb_wrapper/models/pagamentos.py

```diff
@@ -325,43 +388,71 @@ class LiberarPagamentos(BaseModel):
-    325: "Divergência entre o primeiro e último nome do beneficiário VS primeiro e último nome na Receita Federal. Verificar com beneficiário necessidade de ajustes.",  # noqa: E501
+    324: "Agência / Conta do Favorecido Substituída",
+    325: "Divergência entre o primeiro e último nome do beneficiário na Receita Federal",  # noqa
     326: "Confirmação de Antecipação de Valor",
     327: "Antecipação parcial de valor",
     328: "Boleto bloqueado na base. Não passível de pagamento.",
-    329: "Sistema em contingência – Boleto valor maior que referência. Consulte o beneficiário ou tente efetuar o pagamento mais tarde.",  # noqa: E501
-    330: "Sistema em contingência – Boleto vencido. Consulte o beneficiário ou tente efetuar o pagamento mais tarde.",  # noqa: E501
-    331: "Sistema em contingência – Boleto indexado. Consulte o beneficiário ou tente efetuar o pagamento mais tarde.",  # noqa: E501
-    332: "Beneficiário divergente. Verifique dado informado.",
-    333: "Limite de pagamentos parciais do boleto excedido. Consulte o Beneficiário do boleto.",  # noqa: E501
+    329: "Sistema em contingência – Boleto valor maior que referência",
+    330: "Sistema em contingência – Boleto vencido",
+    331: "Sistema em contingência – Boleto indexado",
+    332: "Beneficiário divergente. ",
+    333: "Limite de pagamentos parciais do boleto excedido.",
     334: "Boleto já liquidado. Não passível de pagamento.",
-    999: "Consultar o Banco para detalhar o erro.",
+    999: "Consultar o Banco para detalhar o erro",
+}
+
+ERROOS_ESTORNO_PAGAMENTO_DICT = {
+    1: "Conta destinatária do credito encerrada",
+    2: "Agencia ou conta destinatária do credito invalida",
+    3: "Ausência ou divergência na indicação do CPF/CNPJ",
+    4: "Msg invalida p/ o tipo de transação ou finalidade",
+    5: "Divergência na titularidade",
+    6: "Transferência insuficiente p/ finalidade indicada",
+    7: "Diferença a maior",
+    8: "Código identificador de transferência invalido",
+    9: "Devolução por fraude",
+    15: "Identificação deposito Judicial Invalida",
+    16: "Msg STRO020/PAG0116 fora do horário def do negocio",
+    17: "Numero de contrato invalido",
+    18: "Valor em duplicidade",
+    19: "Mov. finc. ligadas ao terrorismo e seu financiamento",
+    22: "Devolução de ordem bancaria pelo agente financeiro",
+    24: "Erro no Preenchimento do Documento de Recolhimento",
+    25: "Erro no Preenchimento do Deposito Direto",
+    26: "Devolc. de pagmnt de tributos por solicitação da IF",
+    27: "Devolc de recolhimento a maior autorizada pela RFB",
+    28: "Credito não sacado - decurso de prazo estipulado",
+    31: "CPF/CNPJ inapto junto a receita federal do Brasil",
+    61: "Transf supera limite para o tipo de conta destino",
+    70: "Por solicitação de cliente da Inst Partcpt Receb",
+    72: "Não conformidade no pagamento",
+    84: "Ct destino inválida p/ o tp de trans ou finalidade",
```

Comment:
O que `tp de trans` significa?
```suggestion
    84: "Conta destino inválida para o tp de trans ou finalidade",
```